doc/source/user_guide/io.rst

```diff
@@ -3279,6 +3279,32 @@ of reading in Wikipedia's very large (12 GB+) latest article data dump.
 
     [3578765 rows x 3 columns]
 
+The following is an example with ``use_nullable_dtypes=True``:
```

Comment:
Could you move these to the docstring instead of this user guide? (in addition to your other PRs)

Reply:
Will do.

Reply:
@mroeschke I moved the xml and json example but in the ``read_html`` example section it says ```See the :ref:`read_html documentation in the IO section of the docs
    <io.read_html>` for some examples of reading in HTML tables.```

Reply:
It would be good to move those too over to the docstring